### PR TITLE
Feature/per 351

### PR DESCRIPTION
**Pull request by kehahn**

### CODE REVIEW

File: src/server/app.py [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 
-from flask import Flask, request, Response
+from flask import Flask, request, Response, send_from_directory
 from flask_cors import CORS
 
 import src.server.cea_608_encoder.byte_pair_generator as encoder
 
 
-app = Flask(__name__)
+app = Flask(__name__, static_folder='../client/')
```
Comment (mbozic-jama) [inline src/server/app.py]: Does the page load with the static folder argument removed?

File: src/server/app.py [hunk 2]
```diff
@@ -28,12 +33,12 @@ def submit():
                         status=200,
                         mimetype='application/json')
     except IOError as err:
-        app.logger.error(f'Could not write JSON to file: {err}')
+        app.logger.error('Could not write JSON to file: ')
```
Comment (mbozic-jama) [inline src/server/app.py]: The `f` string formatter is heavily supported in python 3. Does this cause you errors when running?

File: src/server/app.py (hunk 1 above)
Comment (kehahn) [reply src/server/app.py]: It still works, removed!

File: src/server/app.py (hunk 2 above)
Comment (kehahn) [reply src/server/app.py]: Not anymore, added it back!

**Comment by JosephJindrich** [comment]
image

This is all I get when I run `flask run` and I get a blank page when I try and open localhost:5000. Everything works when I open the index page manually though.

**Review by JosephJindrich — APPROVED** [review]
I tried this and it works.

**Pull request merged**